### ISSUE TITLE
Update Entities, Traits and Specifications doc to cover `managementPolicy`

### ISSUE THREAD

**Issue by foundrytom** [OpenAssetIO/OpenAssetIO#1055]
## What

Update the traits overview doc to include all uses and help disambiguate.

## Why

We wrote that before we switched `managementPolicy` over to traits, and it's not reflected there.

**Comment by mayo7e** [comment]
i will like to work on this please. thanks

**Comment by mayo7e** [comment]
The project itself is a sort of wrapper on dcc tools or pipeline script that standardizes communication with an Asset management system

An Entity consists of TraitSet which  contains TraitData that describes management policy(a way the host gets feedback from the Manager on interaction preference with different entities and which traits they are capable of resolving)

Updating the entities, traits, and specifications would cover management policy and its use case such as when the requested traits are not present. I am a bit unclear on the use case around the management policy at the moment

**Comment by foundrytom** [comment]
@mayo7e sounds like you have a really good take on it!

In what capacity are you available to help? With docs work, we usually like to try and pop together a high-level sketch of the changes to the structure in something like Google docs (or other collab editor that allows comments), run it by a few people, then add some details, and finally make a PR with the actual changes. The PR cycle tends to get a bit heavy for the initial revisions. We've tried to capture some [user personas](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/doc/contributing/USER_PERSONAS.md) here, to help when writing copy.

**Comment by mayo7e** [comment]
@foundrytom I am  happy to start out contributions with the doc and hopefully take on solving technical problems, I have a background in frontend engineer, using programming tools javascript, rust, and also in UX

**Comment by mayo7e** [comment]
hi @foundrytom can I proceed with this contribution?

**Comment by foundrytom** [comment]
@mayo7e Certainly, do you have an idea for an improved structure for the doc or want to have a sync up about what we see the specific issues being as it stands?

**Comment by mayo7e** [comment]
@foundrytom yes i think a sync up would be great start. there is still some gap in my understanding

**Comment by foundrytom** [comment]
We're in the weeds with a few things this week, and I'm OOO for a few days. Would some time next week work? What time zone are you in? In the mean time, If you're not already familiar with Doxygen, it's worth getting your head around how that works, and how we use it to [build the docs in question](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/doc/doxygen/README.md).

**Comment by mayo7e** [comment]
hi @foundrytom  i am currently on GMT +1, Anytime from next week monday is fine with me. thanks for the direction, i am currently working on getting familar with doxygen in time for our meet  sometime next week. In the interim, resources you have on this or towards understanding the scope for this PR   would be greatly appreciated.

**Comment by mayo7e** [comment]
Hi @foundrytom, how is the week going? trying to follow up on when is suitable  for the brief meet , and if this week is still a possibility..

**Comment by foundrytom** [comment]
Hey @mayo7e, sorry for the delay, this week has vanished! Would Tues 19th work? How are you getting on with Doxygen and such? Are you on ASWF Slack? Might be easier to chat on there or via email (see my profile) to arrange.

**Comment by mayo7e** [comment]
@foundrytom  I will revert via the  #ASWF slack channel
